packages/dashboard-frontend/src/services/workspace-client/devworkspace/devWorkspaceClient.ts

```diff
@@ -745,6 +749,35 @@ export class DevWorkspaceClient extends WorkspaceClient {
     return workspace.metadata.annotations?.[DEVWORKSPACE_DEBUG_START_ANNOTATION] === 'true';
   }
 
+  async updateConfigData(
+    workspace: devfileApi.DevWorkspace,
+    config: api.IServerConfig,
+  ): Promise<devfileApi.DevWorkspace> {
+    const patch: api.IPatch[] = [];
+
+    const devworkspaceConfigPath = `/metadata/annotations/${this.escape(
+      DEVWORKSPACE_CONFIG_ANNOTATION,
+    )}`;
+    const devworkspaceConfig = { name: 'devworkspace-config', namespace: config.cheNamespace };
```

Comment:
@@AObuchow my understanding is that `devworkspace-config` is the name of the DWOC created by the che operator that is not supposed to be changed and we can hardcode it on the UD level, right?

Reply:
smth. we discussed in https://github.com/eclipse/che/issues/21405#issuecomment-1231307953